doc/source/contributing.rst

```diff
@@ -198,12 +196,13 @@ At this point you can easily do a *development* install, as detailed in the next
 3) Installing Dependencies
 --------------------------
 
-To run *geopandas* in an development environment, you must first install
-*geopandas*'s dependencies. We suggest doing so using the following commands
+To run *GeoPandas* in an development environment, you must first install
+*GeoPandas*'s dependencies. We suggest doing so using the following commands
 (executed after your development environment has been activated)::
 
     conda install -c conda-forge fiona shapely pyproj rtree
     conda install pandas
```

Comment:
@jorisvandenbossche is there a reason why pandas is not installed from conda-forge?

Reply:
Not directly an idea. Maybe the initial idea was to only install those things from conda-forge that were needed and the rest from defaults (this document is old ..), but nowadays I would in any case recommend to either all from defaults or all from conda-forge.